tests/test_content_client.py

```diff
@@ -12,8 +12,8 @@ def katello_client(request, entities, registration_hostname):
     container_id = subprocess.check_output([
         'podman',
         'run',
-        '--volume',
-        '/dev/log:/dev/log',
```

Comment:
Can't you make this a parameter somehow? Is this something we can declare in the Dockerfile?

Reply:
You can only define one half of the volume endpoint in the Dockerfile. I can make it a parameter via fixtures most likely if we feel strong enough about it. This work around is only needed for EL6, and being special as is I just felt documenting it was cleaner than legacy code around it (and you also need an EL7 host).

Reply:
Ah right, if the trick only works on an EL7 host I think a parameter might be too complex. You could use variables. We already depend on [pytest-variables](https://github.com/pytest-dev/pytest-variables) so if we want to support it, that would be the way to go. Documenting a setting in a YAML file is probably better than documenting a code change.